### PR TITLE
WRN-5909: Sync up UI-Test and TC.

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Need to change the TC number in the UI-Test

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Change the TC number in the UI-Test

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]:# (Related issues, references)
WRN-5909

### Comments
As the TC moves from Jira to CLM change the TC number in the UI-Test.

### CODE REVIEW

File: tests/ui/specs/SwitchItem/SwitchItem-specs.js [hunk 1]
```diff
@@ -23,12 +23,12 @@ describe.skip('SwitchItem', function () {
 		});
 
 		describe('5-way', function () {
-			it('should select the item when selected - [GT-21377]', function () {
+			it('should select the item when selected', function () {
```
Comment (juwonjeong) [inline tests/ui/specs/SwitchItem/SwitchItem-specs.js:26]: Would it be possible to match TC GT-21377 with TC QWT-2527? 
```suggestion
			it('should select the item when selected - [QWT-2527]', function () {
```

File: tests/ui/specs/TabLayout/VerticalTabsWithIcons/VerticalTabsWithIcons-specs.js [hunk 2]
```diff
@@ -11,8 +11,7 @@ describe('TabLayout', function () {
 		describe('collapsing/expanding behavior', function () {
 			describe('5-way interaction', function () {
 
-				// this covers GT-28257 step 3
-				it('should collapse tabs when focus is moved to a Spottable component in the content container', function () {
+				it('should collapse tabs when focus is moved to a Spottable component in the content container - [QWT-2758]', function () {
```
Comment (juwonjeong) [inline tests/ui/specs/TabLayout/VerticalTabsWithIcons/VerticalTabsWithIcons-specs.js:14]: Can we mention step number on line 18 and 35 instead?
```
// Step 3-2: 5-way right to Spottable component 
// Step 4: Back to the tabs
```

File: tests/ui/specs/TabLayout/VerticalTabsWithoutIcons/VerticalTabsWithoutIcons-specs.js [hunk 3]
```diff
@@ -11,8 +11,7 @@ describe('TabLayout', function () {
 		describe('collapsing/expanding behavior', function () {
 			describe('5-way interaction', function () {
 
-				// this covers GT-28259 step 3
-				it('should collapse tabs when focus is moved to a Spottable component in the content container', function () {
+				it('should collapse tabs when focus is moved to a Spottable component in the content container - [QWT-2756]', function () {
```
Comment (juwonjeong) [inline tests/ui/specs/TabLayout/VerticalTabsWithoutIcons/VerticalTabsWithoutIcons-specs.js:14]: Same as VerticalTabsWithIcons-specs.js
```
// Step 3-2: 5-way right to Spottable component 
// Step 4: Back to the tabs
```

File: tests/ui/specs/SwitchItem/SwitchItem-specs.js (hunk 1 above)
Comment (dongsuwon) [reply tests/ui/specs/SwitchItem/SwitchItem-specs.js:26]: As the QWT-2527 test is the pointer mode test, and this test is a 5-way test, I don't think this test and the test of QWT-2527 are the same. I think the test on 50 line is the same as QWT-2527, what do you think?

Comment (juwonjeong) [reply tests/ui/specs/SwitchItem/SwitchItem-specs.js:26]: Sorry, I forgot to check the test mode:( I think we can just leave it as you fixed it.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**